seisflows/preprocess/pyaflowa.py

```diff
@@ -757,3 +757,10 @@ def _make_evaluation_composite_pdf(self):
             for event_pdf in event_pdfs:
                 os.remove(event_pdf)
 
+
```

Comment:
See comment on `preprocess.py`. I think we can import that function directly from the preprocess script to avoid redundant code.